west.yml

```diff
@@ -32,7 +32,7 @@ manifest:
       revision: 094aeb41bb93e9199d24d665ee43e9e05d6d7b1c
       path: modules/lib/civetweb
     - name: cmsis
-      revision: b0612c97c1401feeb4160add6462c3627fe90fc7
+      revision: pull/14/head
```

Comment:
See comment here:
https://github.com/zephyrproject-rtos/cmsis/pull/14#discussion_r783868889

Reply:
I agree with @carlescufi that this can be done as a follow-up